### PR TITLE
Arman/brb table

### PR DESCRIPTION
**Pull request by arman-esmaili**

### CODE REVIEW

File: Eatery/Controllers/MapViewController.swift [hunk 1]
```diff
@@ -10,16 +10,17 @@ import MapKit
...
     var eateryAnnotations : [MKPointAnnotation] = []
     let mapView: MKMapView
     var locationManager: CLLocationManager!
     
-    let arrowButton = UIButton()
-    let pinButton = UIButton()
+    let recenterButton = UIButton()
+    
+    let defaultLocation = CLLocation(latitude: 42.448078,longitude: -76.484291) // olin library
+    let defaultCoordinate = CLLocation(latitude: 42.448078,longitude: -76.484291).coordinate
```
Comment (anniexcheng) [inline Eatery/Controllers/MapViewController.swift:23]: `defaultLocation.coordinate`

Comment (arman-esmaili) [reply Eatery/Controllers/MapViewController.swift:23]: tried it doesn't compile

File: Eatery/Controllers/BRB Feature/BRBViewController.swift [hunk 2]
```diff
@@ -47,30 +47,30 @@ class BRBViewController: UIViewController, BRBConnectionErrorHandler, BRBLoginVi
         connectionHandler.errorDelegate = self
         
         loginView = BRBLoginView(frame: view.bounds)
-        ai.frame = CGRect(x: view.frame.width/2 - 10, y: 12, width:20, height:20)
-        ai.transform = .init(translationX: 0, y: 10)
-        ai.color = .black
-        ai.hidesWhenStopped = true
+        activityIndicatorView.frame = CGRect(x: view.frame.width/2 - 10, y: 12, width:20, height:20)
```
Comment (anniexcheng) [inline Eatery/Controllers/BRB Feature/BRBViewController.swift]: spaces after `:` for consistency

File: Eatery/Controllers/MapViewController.swift [hunk 3]
```diff
@@ -103,66 +95,30 @@ class MapViewController: UIViewController, MKMapViewDelegate, CLLocationManagerD
...
-        newCamera.pitch = 60
-        newCamera.centerCoordinate = locationManager.location?.coordinate ?? CLLocation(latitude: 42.448078,longitude: -76.484291).coordinate
-        
-        mapView.setCamera(newCamera, animated: true)
+        // Create bottom left re-center button
+        recenterButton.frame = CGRect(x: 20, y: view.frame.size.height - 65, width: 120, height: 40)
+        recenterButton.layer.cornerRadius = 6
+        recenterButton.setImage(UIImage(named: "locationArrowIcon")?.withRenderingMode(.alwaysTemplate), for: .normal)
+        recenterButton.tintColor = UIColor.colorFromCode(0x3d90e2)
+        recenterButton.imageEdgeInsets = UIEdgeInsetsMake(0, -6, 0, 0)
```
Comment (anniexcheng) [inline Eatery/Controllers/MapViewController.swift]: `recenterButton.imageEdgeInsets.left = -6`

File: Eatery/Controllers/MapViewController.swift [hunk 4]
```diff
@@ -103,66 +95,30 @@ class MapViewController: UIViewController, MKMapViewDelegate, CLLocationManagerD
...
+        recenterButton.titleEdgeInsets = UIEdgeInsetsMake(0, 8, 0, 0)
```
Comment (anniexcheng) [inline Eatery/Controllers/MapViewController.swift]: `recenterButton.titleEdgeInsets.left = 8`

**Comment by anniexcheng** [comment]
Good to merge after you make those minor changes :)

File: Eatery/Controllers/MapViewController.swift (hunk 1 above)
Comment (anniexcheng) [reply Eatery/Controllers/MapViewController.swift:23]: oh I see

**Pull request merged**